shared_model/backend/protobuf/impl/proto_query_response_factory.cpp

```diff
@@ -196,6 +196,32 @@ shared_model::proto::ProtoQueryResponseFactory::createTransactionsResponse(
       query_hash);
 }
 
+std::unique_ptr<shared_model::interface::QueryResponse>
+shared_model::proto::ProtoQueryResponseFactory::createTransactionsPageResponse(
```

Comment:
Maybe, the response should be called `transactionsPagedResponse`? More correct from language point of view

Reply:
Depends how to read that:) TransactionsPage Response. i.e. Response with TransactionsPage. I think it is good, isn't it?